site/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <meta name="author" content="Ada">
+    <body>
+        <p>
+            beeeeeeeeeeeeeeeebs
+        </p>
+    </body>
+</head>
```

Comment:
```suggestion
<html>
    <head>
        <meta charset="utf-8">
        <meta name="author" content="Ada">
    </head>
    <body>
        <p>
            beeeeeeeeeeeeeeeebs
        </p>
    </body>
</html>
```
While you don't necessarily need the root `<html>` tag, it is good practice to include it. 

Be cautious with how you nest your html. While the browser may attempt to render content it may lead to unexpected output. If at any point you want to check if your html is valid you can use this [validator](https://validator.w3.org/)